features/functests/performance/performance.go

```diff
@@ -104,19 +105,15 @@ var _ = Describe("performance", func() {
 				mcd, err := mcdForNode(&node)
 				Expect(err).ToNot(HaveOccurred())
 				mcdName := mcd.ObjectMeta.Name
-				By("executing a command inside the pod " + mcdName)
-				out, err := exec.Command("oc", "rsh", "-n", "openshift-machine-config-operator", mcdName,
-					"sysctl", "-A").CombinedOutput()
-				Expect(err).ToNot(HaveOccurred())
-
-				for _, str := range strings.Split(string(out), "\n") {
-					line := strings.Split((string(str)), "=")
-					param := strings.TrimSpace(line[0])
-					value := strings.TrimSpace(strings.Join(line[1:], ""))
-					if expected, ok := sysctlMap[param]; ok {
-						By(fmt.Sprintf("checking whether parameter %s value is %s", param, expected))
-						Expect(value).To(Equal(expected), fmt.Sprintf("parameter %s value is not %s", param, expected))
-					}
+				for param, expected := range sysctlMap {
+					By(fmt.Sprintf("executing the command \"sysctl -n %s\" inside the pod %s", param, mcdName))
+					Eventually(func() string {
+						out, err := exec.Command("oc", "rsh", "-n", mcd.ObjectMeta.Namespace,
```

Comment:
looks like the new format of the command is now failing and getting an error upon execution